### PR TITLE
Migrate off ovh rise-1 server (to kubernetes)

### PR DESCRIPTION
**Pull request by cmyui**

### CODE REVIEW

File: app/common/views.py [hunk 1]
```diff
@@ -232,7 +229,7 @@ async def callback(self, interaction: discord.Interaction):
         await sw_requests.partial_update(
             request_data["score_id"],
             status.value,
-            datetime.datetime.utcnow(),
+            datetime.datetime.now(datetime.UTC),
```
Comment (cmyui) [inline app/common/views.py:232]: does this work? i remember needing timezone.utc but not sure what datetime.UTC is

Comment (cmyui) [reply app/common/views.py:232]: image

File: app/osu_beatmaps.py [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+) -> Image.Image | None:
+    osu_background_url = f"https://api.osu.direct/media/background/{beatmap_id}"
+    response = await state.http_client.get(
+        osu_background_url,
+        headers={"User-Agent": "akatsuki/management-bot"},
+    )
+    response.raise_for_status()
+
+    background_data = response.read()
+    if b"beatmap not found!" in background_data:
```
Comment (cmyui) [inline app/osu_beatmaps.py:78]: do they send back a 404?

File: app/osu_replays.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+
+class Replay(ReplayFile):
+    raw_replay_data: bytes
+
+
+async def get_replay(score_id: int) -> Replay | None:
+    response = await state.http_client.get(
+        f"https://akatsuki.gg/web/replays/{score_id}",
+        headers={"User-Agent": "akatsuki/management-bot"},
```
Comment (cmyui) [inline app/osu_replays.py:17]: should the user agent be set on the http client instantiation?

File: app/main.py [hunk 4]
```diff
@@ -115,11 +102,21 @@ async def on_ready() -> None:
     )
     await state.write_database.connect()
 
-    state.http_client = http.HTTPClient()
+    state.http_client = httpx.AsyncClient(
+        follow_redirects=True,
+        timeout=60,
+    )
```
Comment (cmyui) [inline app/main.py:108]: not an expectation for this pr but in general this should _proobably_ be split into more http clients on a per-vendor basis, as rules like timeouts, user agents, rate limit configuration, etc. are often different per vendor

**Review by lenforiee — APPROVED** [review]

**Pull request merged**